CMakeLists.txt

```
@@ -80,6 +80,9 @@ set(WITH_LFORTRAN_BINARY_MODFILES YES
 set(WITH_RUNTIME_LIBRARY YES
     CACHE BOOL "Compile and install the runtime library")
 
+set(WITH_INTRINSIC_MODULES no
+    CACHE BOOL "Compile intrinsic modules till ASR level at build time")
```

I would maybe write it as:
```suggestion
    CACHE BOOL "Compile intrinsic modules to .pyc (ASR) at build time")
```
But that can be done later.